opennms-config/src/main/java/org/opennms/netmgt/config/DefaultEventConfDao.java

```diff
@@ -164,7 +164,8 @@ public void saveCurrent() {
 				accum.add(event);
 				return accum;
 			}
-		});
+            // remove duplicates
```

Comment:
Can we elaborate on why we remove the duplicates?

Reply:
I added a comment - basically because we roll up the prioritized events and they still remain in thier original position as well, we would check the event multiple times.
This method is only used for testing at this time so I could revert the change if you prefer.